app/experimenter/experiments/constants.py

```diff
@@ -1231,9 +1231,5 @@ def FEATURE_TYPE_CHOICES(cls):  # pragma: no cover
 
 {experiment.experiment_url}
         """
-    BUGZILLA_RAPID_EXPERIMENT_TEMPLATE = """ This is an empty CFR A/A experiment.
-     The A/A experiment is being run to test the automation, effectiveness,
-     and accuracy of the rapid experiments platform.
-     The experiment is an internal test,
-     and Firefox users will not see any noticeable change
-     and there will be no user impact."""
+    BUGZILLA_RAPID_EXPERIMENT_TEMPLATE = """ This is an empty CFR A/A experiment.The A/A experiment is being run to test the automation, effectiveness, and accuracy of the rapid experiments platform.
```

Comment:
nit: add space between the "experiment." and "The A/A"

Reply:
eek, lemme fix that, thanks @pdehaan